docs/api/api-spec.md

```diff
@@ -170,10 +170,14 @@ To find a system channel, one calls
 To join a channel. one calls
 
 ```javascript
-    fdc3.joinChannel(redChannel.id);
+    redChannel.join();
+
+    //join the 'default' channel to effectively reset channel membership
+    const defaultChannel = await fdc3.getOrCreateChannel('default');
```

Comment:
This example seems to counter the original documentation for `getOrCreateChannel(...)` which describes them as "these custom channels" and presumably set apart from the predefined system channels returned by `fdc3.getSystemChannels()`.

Reply:
with the new PR, there is no class differentiation between system and app channels - just  convention based.  While call 'getOrCreateChannel' for a system channel isn't necessary, it doesn't hurt anything either.